jbpm-test/src/main/resources/jBPM.properties

```diff
@@ -1,9 +1,9 @@
 # for creating a datasource
-persistence.datasource.name=jdbc/jbpm-ds
-persistence.datasource.user=sa
-persistence.datasource.password=
-persistence.datasource.url=jdbc:h2:tcp://localhost/~/jbpm-db
-persistence.datasource.driverClassName=org.h2.Driver
+name=jdbc/jbpm-ds
+driverClassName=org.h2.Driver
+user=sa
+password=
+url=jdbc:h2:tcp://localhost/~/jbpm-db
```

Comment:
I still think these should be left as before, since JBPMHelper still uses format with `persistence.datasource` prefix. See:

https://github.com/kiegroup/jbpm/blob/5dfd109a778a9cde753302d527418655556154e9/jbpm-test/src/main/java/org/jbpm/test/JBPMHelper.java#L97-L102

Can we leave it as it was before? And after that in future we can remove JBPMHelper completely?

Reply:
ok. feel free to file a jira regarding removing the JBPMHelper...